runtime/compiler/env/j9method.cpp

```diff
@@ -2887,6 +2888,7 @@ void TR_ResolvedJ9Method::construct()
       {x(TR::sun_misc_Unsafe_putFloatVolatile_jlObjectJF_V,         "putFloatRelease",   "(Ljava/lang/Object;JF)V")},
       {x(TR::sun_misc_Unsafe_putDoubleVolatile_jlObjectJD_V,        "putDoubleRelease",  "(Ljava/lang/Object;JD)V")},
       {x(TR::sun_misc_Unsafe_putObjectVolatile_jlObjectJjlObject_V, "putObjectRelease",  "(Ljava/lang/Object;JLjava/lang/Object;)V")},
+      {x(TR::sun_misc_Unsafe_putObjectVolatile_jlObjectJjlObject_V, "putReferenceRelease",  "(Ljava/lang/Object;JLjava/lang/Object;)V")},
 
```

Comment:
what is the motivation to keep the enum name of `TR::sun_misc_Unsafe_putObjectVolatile_jlObjectJjlObject_V`  for method `putReferenceRelease`?

Reply:
I was following the convention that was already being used in the area. For example one line up the existing `putObjectRelease` also reuses `TR::sun_misc_Unsafe_putObjectVolatile_jlObjectJjlObject_V`.

`putObjectRelease` does nothing but call `putObjectVolatile`.
`putReferenceRelease` does nothing but call `putReferenceVolatile`.

`putObjectRelease`, `putObjectVolatile` and `putReferenceVolatile` are all recognized as `TR::sun_misc_Unsafe_putObjectVolatile_jlObjectJjlObject_V`. So it seemed to me that `putReferenceRelease` should also do the same.

Reply:
ok.  for these particular methods, this might be the right way to go.  otherwise, more works are required to optimize them, since their bytecode(s) contain another call.